game/scripts/vscripts/util.ts

```diff
@@ -526,6 +569,19 @@ export function DirectionToPosition(origin_pos: Vector, towards_pos: Vector): Ve
     return ((towards_pos - origin_pos) as Vector).Normalized();
 }
 
+export function removeNeutralSpawners() {
+    //const creepSpawnLocationToKeep = Entities.FindAllByClassnameWithin("npc_dota_neutral_spawner",Vector(-2464,4816,170),500)[0];
+    const spawners = Entities.FindAllByClassname("npc_dota_neutral_spawner");
+    for (const spawner of spawners)
+    {
+        let normal = spawner.GetAbsOrigin()-Vector(-2464,4816,170) as Vector
+        if (normal.Length2D() > 100) {
+            UTIL_Remove(spawner);
+        }
+        
+    }
+}
+
```

Comment:
Why is this function in here?

Reply:
Why not? I also think that should be in util, it's not section specific.

Also, formatting.

Reply:
I mean, it seems completely unrelated to this PR so it looks like a mistake?